db/seeds.rb

```diff
@@ -67,7 +67,6 @@
 
 # Add some default companies and add relationship to test user
 companies.each do |company|
-  puts company[:email]
```

Comment:
This was there so that one can be sure the seed script ran.

Reply:
It's a debugging tool, which is why I removed it.